.github/workflows/terraform-observe_scheduler.yaml

```
@@ -83,4 +83,4 @@ jobs:
           cmd: yq '.iconUrl' manifest.yaml
       - name: Validate App Icon URL
         if: ${{ !startsWith(steps.get_icon_url.outputs.result, 'https://assets.observeinc.com/') && steps.check_manifest.outputs.files_exists == 'true' }}
-        run: echo "::error file=mainfest.yaml, title=Invalid App Icon URL::App icon URLs must live in the 'assets.observeinc.com' domain" && exit 1
+        run: echo "::error file=manifest.yaml, title=Invalid App Icon URL::App icon URLs must live in the 'assets.observeinc.com' domain" && exit 1
```

```suggestion
        run: echo "::error file=manifest.yaml,title=Invalid App Icon URL::App icon URLs must live in the 'assets.observeinc.com' domain" && exit 1
```

The documentation indicates no spacing between key/value pairs in an a workflow command. It may still work but it's not worth leaving to chance. After the main delimiter (`::`) you should be fine to have whitespace.